### PR TITLE
Changelog for v3.0.0

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
 
+v3.0.0 / 2019-04-08
+===================
+
+* [Feature - Breaking] Introduce TestLoadBalancing <@choheekim> & <step2yeung>
+
+Starting with version `3.0.0`, Ember-exam's `start()` function must be invoked explicitly to use ember-exam's functionalities. Prior to this release, Ember Exam must be loaded by importing `addon-test-support/load.js` and calling `loadEmberExam`. Ember-Exam's `start()` function ensures tests are loaded at the right places and unifies to set up for both `ember-qunit` and `ember-mocha`. If you are using ember-exam version < `3.0.0` make sure to invoke `loadEmberExam()`.
```
Comment (stefanpenner) [inline CHANGELOG.md]: @step2yeung can you provide a brief description as to why this change was made, and also what it buys us?

Comment (step2yeung) [reply CHANGELOG.md]: Prior to ember-exam v3, ember-exam setup it's functionality by monkey patching either ember-qunit or ember-mocha's test-loader.
In ember-exam v3, instead of monkey patching, we use an instance of the test loader which extend the test-loader with ember-exam's functionality. Hence requiring start()to utilize the new test loader.

**Comment by step2yeung** [comment]
looks good! Thanks for providing an clear changlog entry.
